scripts/schmidt_comparison.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.insert(0,'../chamber/models')
+
+from CoolProp import HumidAirProp as hap
+import pandas as pd
+
+import props
+
+
+results = {'Pressure (Pa)': [], 'Temperature (K)': [],
+					'Ralative Humidity': [], 'Schmidt Mills': [],
+					'Schmidt Marrero': []}
+
+
+# Gets relevant values and calculates Schmidt number
+def get_schmidt(p, t, rh):
+	mills = props.get_d_12(p, t, 'Mills')
+	marrero = props.get_d_12(p, t, 'Marrero')
+
+	# Sometimes errors arise with coolprop from high relative humidities
+	try:
+		rho = 1/hap.HAPropsSI('Vha', 'P', p, 'T', t, 'RH', rh)
+	except ValueError:
+		return False, False
+
+	mu = props.get_mu(p, t, rh)
+	schmidt_mills = mu/(rho*mills)
+	schmidt_marrero = mu/(rho*marrero)
+
+	return schmidt_mills, schmidt_marrero
+
+
+if __name__=="__main__":
+	for press in range(30000,100001,10000):
+		# Uses a line approximating the model limit curve to limit the upper range
+		for temp in range(275, int(press*0.0005+261), 5):
```

Comment:
Rather than use this linear approximation for the model limit, use the model. It is in the paper, and we can talk about it more, but what you are looking for is something like `for temp in range(275, 310*pow(press/101325, 0.09), 5):` rather than what you have. But in general, this was a smart way to approach the problem.

See [here](https://www.desmos.com/calculator/x3bvcyhmqb)